### PR TITLE
Fix build of documentation in libstd

### LINKED ISSUE #603: Building as rust submodule is broken
See https://travis-ci.org/rust-lang/rust/jobs/458312853 and https://github.com/rust-lang/rust/pull/56178

Probably caused by @japaric's changes in #558.

### PR DESCRIPTION
**Pull request by alexcrichton**
Ensure we've always got a macro defined!

Closes #603

### CODE REVIEW

**Comment by jethrogb** [comment]
Nope https://travis-ci.org/rust-lang/rust/jobs/460618617

File: coresimd/arm/armclang.rs [hunk 1]
```diff
@@ -39,6 +39,11 @@ use stdsimd_test::assert_instr;
 #[inline(always)]
 #[rustc_args_required_const(0)]
 pub unsafe fn __breakpoint(val: i32) {
+    // Ensure that this compiles correctly on non-arm architectures, so libstd
+    // doc builds work. The proper macro will shadow this definition below.
+    #[allow(unused_macros)]
+    macro_rules! call { ($e:expr) => {} }
```
Comment (gnzlbg) [inline coresimd/arm/armclang.rs]: This might need to be `=> $e` ? @jethrogb  ?

**Comment by jethrogb** [comment]
Huzzah! https://travis-ci.org/rust-lang/rust/builds/461184580

PS @alexcrichton please add a comment when pushing your branch, GitHub doesn't send an email automatically.

**Comment by gnzlbg** [comment]
LGTM, @alexcrichton this can be merged right?

**Comment by jethrogb** [comment]
The rustfmt CI failure is real

**Pull request merged**